python/tests/experimental/extras/test_embedding_metric.py

```diff
@@ -17,7 +18,7 @@ def test_embedding_metric_holds_the_smoke_in() -> None:
         labels=["B", "A"],
         distance_fn=DistanceFunction.euclidean,
     )
-    schema = DeclarativeSchema([ResolverSpec(column_name="col1", metrics=[MetricSpec(EmbeddingMetric, config)])])
+    schema = DeclarativeSchema([ResolverSpec(column_type=Tensor, metrics=[MetricSpec(EmbeddingMetric, config)])])
```

Comment:
I think resolving by type only works for rows, Pandas will just call tensor columns `object`.